src/transformStateWithClones.js

```diff
@@ -66,6 +66,29 @@
  */
 function transformStateWithClones(state, transforms) {
   // write code here
+  const stateArr = [];
```

Comment:
```suggestion
  const clonedStates = [];
```
Choose meaningful names